### PR TITLE
Small fixes on tests

### PR DESCRIPTION
**Pull request by kaapstorm**
## Technical Summary

Tests only: Fixed a few weird things I encountered. See commit messages for descriptions. :blowfish: 

### Safety story

Only tests are changed

### Automated test coverage

Yes

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/motech/repeaters/tests/test_models.py [hunk 1]
```diff
@@ -229,38 +229,45 @@ def set_next_attempt_at(repeater, when):
...
     pass
 
 
 class IsResponseTests(SimpleTestCase):
 
     def test_has_text(self):
-        resp = ResponseMock()
-        resp.text = '<h1>Hello World</h1>'
+        resp = Unknown()
+        resp.text = '<h1>Hello World</h1>'  # type: ignore
```
Comment (millerdev) [inline corehq/motech/repeaters/tests/test_models.py:240]: What does `# type: ignore` do? I assume it's a type hint, but do we have any automation that cares about that?

File: corehq/motech/repeaters/tests/test_models.py [hunk 2]
```diff
@@ -229,38 +229,45 @@ def set_next_attempt_at(repeater, when):
...
+class Unknown:
     pass
 
 
 class IsResponseTests(SimpleTestCase):
 
     def test_has_text(self):
-        resp = ResponseMock()
-        resp.text = '<h1>Hello World</h1>'
+        resp = Unknown()
```
Comment (millerdev) [inline corehq/motech/repeaters/tests/test_models.py:239]: Why was this changed from `ResponseMock` (which looks like it was moved below) to `Unknown`?

**Review by millerdev — COMMENTED** [review]
I assume the tests were passing without these changes. Is this mainly cleanup/refactoring?

File: corehq/motech/repeaters/tests/test_models.py (hunk 1 above)
Comment (kaapstorm) [reply corehq/motech/repeaters/tests/test_models.py:240]: `# type: ignore` tells IDEs and mypy not to give the developer type warnings -- in this case specifically not to warn them that they are assigning a value to an undefined attribute. It is the equivalent of `# flake8: noqa`.

File: corehq/motech/repeaters/tests/test_models.py (hunk 2 above)
Comment (kaapstorm) [reply corehq/motech/repeaters/tests/test_models.py:239]: `corehq/motech/repeaters/tests/test_models.py` still defines and uses a `ResponseMock` class, which mocks a Response object. I added the `Unknown` class because it does not mock a Response object.

**Comment by kaapstorm** [comment]
Sorry, I should have added more clarification in the PR description.

> I assume the tests were passing without these changes. Is this mainly cleanup/refactoring?

Yes, the tests were passing. This PR is a cleanup that follows up on feedback from mypy.

File: corehq/motech/repeaters/tests/test_models.py (hunk 1 above)
Comment (millerdev) [reply corehq/motech/repeaters/tests/test_models.py:240]: Could it be avoided by adding `text = ""` to the `Unknown` class?

Comment (kaapstorm) [reply corehq/motech/repeaters/tests/test_models.py]: Comments dropped in 68e39da35de

> Could it be avoided by adding `text = ""` to the `Unknown` class?

That would resolve the warnings, but it would cause some of the tests for `is_response()` to fail, because it would make the instance of `Unknown` look like a response.

**Review by millerdev — APPROVED** [review]
